compiler/interface.go

```diff
@@ -461,6 +461,7 @@ func (c *Compiler) getInterfaceInvokeWrapper(f *ir.Function) llvm.Value {
 	paramTypes := append([]llvm.Type{c.i8ptrType}, fnType.ParamTypes()[len(expandedReceiverType):]...)
 	wrapFnType := llvm.FunctionType(fnType.ReturnType(), paramTypes, false)
 	wrapper = llvm.AddFunction(c.mod, wrapperName, wrapFnType)
+	wrapper.LastParam().SetName("parentHandle")
```

Comment:
Can you add a check whether the to-be-wrapped function also has this name? That would protect against future changes that might break this assumption.

```suggestion
	if f.LLVMFn.LastParam().Name() == "parentHandle" {
		wrapper.LastParam().SetName("parentHandle")
	}
```